single-sign-on/index.md

```diff
@@ -18,6 +18,14 @@ To enable SSO in Docker Hub, you need the following information from your identi
 
 We currently support enabling SSO on a single organization. However, we do not support single logout. If you have any users in your organization with a different domain (including social domains), they will be added to the organization as guests. Guests will continue to authenticate through Docker with their Docker login credentials (Docker ID and password).
 
+
+## Single Sign-on architecture flow
+
+The following diagram shows how Single Sign-on (SSO) operates and is managed in Docker Hub and Docker Desktop. In addition, it provides information on how to authenticate between your IdPs.
+
+![SSO architecture](images/sso-architecture.png)
```

Comment:
@jerae-duffin The following syntax creates a zoomable version of the image. Tried this by cloning your PR branch and building docs locally on my machine. Let me know if this works for you.

```[![SSO architecture](images/sso-architecture.png)](images/sso-architecture.png)```

Reply:
Wondering if we can also include `{: target="_blank" rel="noopener" class="_"}` so the image opens in a new tab? Without this option, if a user clicks the `X`  button to close the zoomed in image, it closes the browser session

Reply:
Updated! Thanks so much!